### DISCUSSION TITLE
1/1/25 Requirements: Which Drug Unit and Type Should Be Listed?

### DISCUSSION THREAD

**Discussion by kntpricing** [CMSgov/hospital-price-transparency#136] [Q&A] [answered]
For the January 1, 2025 CMS requirements of drug_unit_of_measurement and drug_type_of_measurement, are we meant to post the information based on the HCPCS code (if applicable) or the dispense unit and type that is specific to that drug in our formulary? An example is HCPCS code J9211. The HCPCS code description includes unit and type of "5 mg" however we have three drugs in our formulary which are dispensed at 1mg/mL (RXDispenseUnit: mL).

EXAMPLE:

### J9211

**HCPCS Dispense Unit:**
Injection, idarubicin hydrochloride, 5 mg

**Formulary Dispense Unit:**
IDAMYCIN PFS 1 MG/ML INTRAVENOUS SOLUTION

**Comment by RidenourK** [comment, marked as answer]
@kntpricing. Thank you for your question. 

In the CY 2024 OPPS/ASC Final Rule, we reiterated that we had seen hospital MRFs in which the drug unit and type of measurement were either not specified or were included in the same field as the description of the item or service… when the drug unit and type of measurement is not specified, the user of the file has no basis for understanding the standard charge that the hospital has established. In other words, the description is not sufficient for the user to understand **what quantity of the item or service the user would receive at the indicated standard charge amount.** 

We also indicated that **the drug unit and type of measurement are intended to bring context to the standard charge a hospital has established for the drug, which typically (but may not always be) expressed as a dose, leveraging HCPCS or NDC dosing descriptions.** Please refer to the discussion at [88 FR 82102](https://www.federalregister.gov/d/2023-24293/p-4997)-[88 FR 82103](https://www.federalregister.gov/d/2023-24293/p-5007). 

Please refer to the CSV Wide examples, the CSV Tall examples, or the JSON examples available on this repository for [examples](https://github.com/CMSgov/hospital-price-transparency/tree/master/examples) of how you could encode drug unit and drug type of measurement.
